core/src/test/java/org/frankframework/http/rest/OpenApiTest.java

```diff
@@ -7,19 +7,12 @@
 import org.frankframework.testutil.MatchUtils;
 import org.frankframework.testutil.ParameterBuilder;
 import org.frankframework.testutil.TestFileUtils;
-import org.frankframework.testutil.threading.IsolatedThread;
-import org.frankframework.testutil.threading.RunInThreadRule;
-import org.junit.Rule;
 import org.junit.jupiter.api.Test;
 import org.springframework.mock.web.MockHttpServletRequest;
 
 public class OpenApiTest extends OpenApiTestBase {
 
-	@Rule
-	public RunInThreadRule runInThread = new RunInThreadRule();
-
 	@Test
-	@IsolatedThread
```

Comment:
Blijkbaar is het voor deze tests niet nodig om ze in een los thread te draaien maar mocht het nodig zijn, dan heeft JUnit5 er ook een `@Isolated` annotation voor.
Ik verwacht dat die ongeveer hetzelfde zal doen als deze custom annotation?

Reply:
Ja dat is vooral handig bij het parallel draaien van tests, ik denk dat we daar zodra alles over is naar JUnit5 is goed naar moeten kijken, volgens mij kunnen we best wat 'test'-tijd besparen namelijk.